Mathlib/Topology/UnitInterval.lean

```diff
@@ -160,6 +224,19 @@ theorem nonneg' {t : I} : 0 ≤ t :=
 theorem le_one' {t : I} : t ≤ 1 :=
   t.2.2
 
+@[simp]
+lemma ne_zero_iff_pos {x : I} : x ≠ 0 ↔ 0 < x := by
```

Comment:
Hmm, there's some `pos_iff_ne_zero` lemmas already; you might match those. Does this need to be simp, though? lt doesn't obviously seem simpler than ne

Reply:
I think there is argument why lt is more useful, but I won't fight for that. Changed it to `pos_iff_ne_zero` and removed the simp attribute.